pygeoapi/provider/mvt_elastic.py

```diff
@@ -162,11 +164,17 @@ def get_tiles(self, layer=None, tileset=None,
             else:
                 url_query = ''
 
-            with requests.Session() as session:
-                session.get(base_url)
-                resp = session.get(f'{base_url}/{layer}/{z}/{y}/{x}{url_query}')  # noqa
-                resp.raise_for_status()
-                return resp.content
+            try:
+                with requests.Session() as session:
+                    session.get(base_url)
+                    resp = session.get(f'{base_url}/{layer}/{z}/{y}/{x}{url_query}')  # noqa
+                    resp.raise_for_status()
+                    return resp.content
+            except requests.exceptions.RequestException as e:
+                LOGGER.debug(e)
+                if resp.status_code <= 500:
```

Comment:
I think `< 500` is meant here instead of `<= 500`, otherwise internal server errors would be reported as invalid client errors

Reply:
Indeed. Well spotted!

Reply:
@totycro fixed on my latest commit